src/routes/MapExplorer/MapExplorer.store.js

```diff
@@ -0,0 +1,395 @@
+import { get } from 'lodash/fp'
+import { createSelector } from 'reselect'
+import bboxPolygon from '@turf/bbox-polygon'
+import booleanWithin from '@turf/boolean-within'
+import { point } from '@turf/helpers'
+import { FETCH_MEMBERS_MAP, FETCH_POSTS_MAP } from 'store/constants'
+import { POST_TYPES } from 'store/models/Post'
+import postsQueryFragment from 'graphql/fragments/postsQueryFragment'
+import { makeGetQueryResults, makeQueryResultsModelSelector } from 'store/reducers/queryResults'
+
+export const MODULE_NAME = 'MapExplorer'
+export const STORE_FETCH_POSTS_PARAM = `${MODULE_NAME}/STORE_FETCH_POSTS_PARAM`
+export const STORE_CLIENT_FILTER_PARAMS = `${MODULE_NAME}/STORE_CLIENT_FILTER_PARAMS`
+
+export const SORT_OPTIONS = [
+  { id: 'updated', label: 'Latest' },
+  { id: 'votes', label: 'Popular' }
+]
+
+export const FEATURE_TYPES = {
+  ...POST_TYPES,
+  member: {
+    primaryColor: '#2A4059', // $color-member
+    backgroundColor: '#FAFBFC', // $color-athens-gray
+    map: true
+  }
+}
+
+const communityPostsQuery = `query (
```

Comment:
It could be nice to move these graphql queries out of this local store and into the `src/graphql`. I could go either way, but in terms of moving the project over time to a more idiomatic leveraging of graphql I started moving queries out these local "pod" `.store.js` files... It's again a longer discussion, if we ever want to dip into it... and this is all just fine!

Reply:
interesting, seems like at that point we may just want to go full apollo... :)